addons/purchase_stock/tests/test_average_price.py

```diff
@@ -3,6 +3,7 @@
 from odoo.addons.stock_account.tests.test_anglo_saxon_valuation_reconciliation_common import ValuationReconciliationTestCommon
 from odoo.tests import tagged, Form
 
+from datetime import date
```

Comment:
you can just use  time.strftime('%Y-%m-%d') for the bill date to reamin consistent with the other tests